File: Pattern25.py
# 1
# 0 1
# 1 0 1
# 0 1 0 1
# 1 0 1 0 1

n = int(input("Enter the number of rows:"))
for i in range(1,n+1):
    start = 0 if i%2==0 else 1
    for j in range(1,i+1):
        print(start, " ",end='')
        start = 0 if start==1 else 1
    print()

File: Pattern25.java
import java.util.Scanner;

/**
 * Pattern25
 */
public class Pattern25 {

    public static void main(String[] args) {
        System.out.println("enter number:");
        Scanner inp = new Scanner(System.in);
        int n = inp.nextInt();
        for (int i = 1; i <= n; i++) {
            for (int j = i; j >= 1; j--) {
                
                System.out.print(j%2);

            }
            System.out.println();

        }
    }
}